meta-mender-tegra/recipes-bsp/u-boot/libubootenv-fake_1.0.bb

```diff
@@ -18,4 +18,4 @@ do_install() {
 }
 
 PACKAGE_ARCH = "${MACHINE_ARCH}"
-RCONFLICTS_${PN} = "u-boot-fw-utils"
+RCONFLICTS_${PN} = "libubootenv"
```

Comment:
I think this needs some tweaking to align with how the OE-Core libubootenv recipe packages the fw utilities.

Reply:
Fixed with 076fd14ff91e38a2fb219ccf70b36816fe9b7a56